.github/workflows/compiler-zoo.yml

```diff
@@ -12,7 +12,45 @@ on: [push]
 permissions:
   contents: read
 
+# This has to be a separate job, it seems, because we want to use a
```

Comment:
A thought: would this be better placed in the os-zoo.yml file?

It seems to be more about Alpine than a compiler.  If not now, then in the future when others have caught up.

Reply:
Ah, good idea!